src/common/src/array/data_chunk.rs

```diff
@@ -725,6 +725,17 @@ impl DataChunkTestExt for DataChunk {
             if let Some(s) = s.strip_suffix("[]") {
                 return DataType::List(Box::new(parse_type(s)));
             }
+
+            if let Some(inner) = s.strip_prefix("map<").and_then(|s| s.strip_suffix('>')) {
```

Comment:
would you mind leaving a comment for the hacks for the `map` and link to the test cases in `src/expr/impl/src/scalar/map_filter.rs`, just make sure that other maintainers can get the idea quickly when looking into this segment